### PR TITLE
fix: Swap-2552, Printing in acrobate reader

### PR DESCRIPTION
**Pull request by bolmsten**
## Description

Removed annotation links in the index file and cleaned up unused files

## Motivation and Context

Fix printing in acrobate reader

## How Has This Been Tested

locally

## Fixes

Swap-2552

### CODE REVIEW

**Review by jekabs-karklins — APPROVED** [review]
✅

File: src/pdf/pdfTableOfContents/index.ts [hunk 1]
```diff
@@ -39,14 +32,10 @@ export function createToC(inFile: string, outFile: string, origOutline: any[]) {
     translatePageNumbers(parser, childOutline, howManyPages)
   );
 
+  console.log(translatedOutline);
```
Comment (martin-trajanovski) [inline src/pdf/pdfTableOfContents/index.ts:35]: You should clean this console.log

**Review by martin-trajanovski — CHANGES_REQUESTED** [review]

File: src/pdf/pdfTableOfContents/index.ts (hunk 1 above)
Comment (jekabs-karklins) [reply src/pdf/pdfTableOfContents/index.ts:35]: > You should clean this console.log

👍🏻

File: src/pdf/pdfTableOfContents/index.ts [hunk 2]
```diff
@@ -1,24 +1,17 @@
...
   // Start new PDF to contain TOC pages only
   const newPDFWriter = muhammara.createWriter(outFile);
-  // const outlineSize = countOutline(origOutline);
-  // const howManyPages = countPages(outlineSize);
-  const howManyPages = 1;
-  const tocText = getTOCText(origOutline, howManyPages);
-  //const tocPageSize = addTOCPages(newPDFWriter, tocText, font)
+  const outlineSize = countOutline(origOutline);
+  const howManyPages = countPages(outlineSize);
+  // const howManyPages = 1;
```
Comment (ACLay) [inline src/pdf/pdfTableOfContents/index.ts:13]: Can this commented out code be removed too?

**Review by martin-trajanovski — APPROVED** [review]

**Pull request merged**